kyaml/openapi/openapi.go

```diff
@@ -72,16 +73,15 @@ func AddSchemaFromFile(path string) error {
 
 // DeleteSchemaInFile reads the file at path and removes all the openAPI definitions
 // present in file from global schema
-func DeleteSchemaInFile(openAPIPath string) error {
+func DeleteSchemaInFile(openAPIPath string) {
 	fields, err := DefinitionRefs(openAPIPath)
 	if err != nil {
-		return err
+		log.Fatal(err.Error())
```

Comment:
We should return an error instead of crashing here.

Reply:
I tried that but lint catches that. 

```
setters2/list.go:35:34: Error return value of `openapi.DeleteSchemaInFile` is not checked (errcheck)
	defer openapi.DeleteSchemaInFile(openAPIPath)
```